src/map/skill.c

```diff
@@ -5650,8 +5661,8 @@ static int skill_castend_id(int tid, int64 tick, int id, intptr_t data)
 		}
 
 		if(ud->skill_id == RG_BACKSTAP) {
-			uint8 dir = map->calc_dir(src,target->x,target->y),t_dir = unit->getdir(target);
-			if(check_distance_bl(src, target, 0) || map->check_dir(dir,t_dir)) {
+			enum unit_dir dir = map->calc_dir(src,target->x,target->y), t_dir = unit->getdir(target);
```

Comment:
* Missing spaces in argument list: `map->calc_dir(src, target->x, target->y)`
* Please declare each variable in its own line.

Reply:
done.